### ISSUE TITLE
CMake Logic to find Arduino SDK

### ISSUE THREAD

**Issue by legounicycler** [SterlingPeet/fprime-arduino#6]
In the toolchain cmake file, there is a comment saying "Check if ARDUINO_SDK_PATH is set, otherwise set it to /opt/arduino-1.8.15/", but there is not actually any logic that checks if the path is set. In fact it currently searches for the toolchain folder somewhere else entirely. For the GT-2 development process, this means we have to manually change this path to the mentioned path in the comment. This is a small change, but it would be nice to see this logic integrated, particularly so that as we pull updates for this submodule there will not be any local changes on our machine.

The lines of interest are as follows:
https://github.com/SterlingPeet/fprime-arduino/blob/379c1ff8110fc06a32179ddd914f29a776b39e6e/cmake/toolchain/Arduino.cmake#L11-L12

**Comment by SterlingPeet** [comment]
yes, this needs to be fixed/addressed, it hasn't because a lot of things in this file should be in the platform file instead.  So motivation to fix something and them move and debug it again, tends to be low.
